@LinProp/LinProp.m

```diff
@@ -1,42 +1,55 @@
+%   <a href="matlab:help LinProp.get_idof -displayBanner"               >get_idof</a>                Inverse degrees of freedeom
+%   <a href="matlab:help LinProp.get_jacobi -displayBanner"             >get_jacobi</a>              Sensitivities to virtual base inputs
+%   <a href="matlab:help LinProp.get_jacobi2 -displayBanner"            >get_jacobi2</a>             Sensitivities to intermediate results
+%   <a href="matlab:help LinProp.get_moment -displayBanner"             >get_moment</a>              n-th central moment
+%   <a href="matlab:help LinProp.get_unc_component -displayBanner"      >get_unc_component</a>       Uncertainty components of y with respect to x
+%   <a href="matlab:help LinProp.unc_budget -displayBanner"             >unc_budget</a>              Opens budget window (LinProp only)
 %
-% LinProp Binary File
-% a = LinProp(filepath, 'binary_file')
+% <strong>Interpolation and Integration Methods</strong>
+%   <a href="matlab:help LinProp.integrate -displayBanner"          >integrate</a>           Integration with cumulative result
+%   <a href="matlab:help LinProp.integrate2 -displayBanner"         >integrate2</a>          Integration with scalar result
+%   <a href="matlab:help LinProp.interpolation -displayBanner"      >interpolation</a>       Interpolation
+%   <a href="matlab:help LinProp.interpolation2 -displayBanner"     >interpolation2</a>      Interpolation with linear unc. propagation
+%   <a href="matlab:help LinProp.spline -displayBanner"             >spline</a>              Spline interpolation
+%   <a href="matlab:help LinProp.spline2 -displayBanner"            >spline2</a>             Spline interpolation with linear unc. propagation
+%   <a href="matlab:help LinProp.splinecoefs -displayBanner"        >splinecoefs</a>	
+%   <a href="matlab:help LinProp.splineintegrate -displayBanner"    >splineintegrate</a>     Spline integration with cumulative result
+%   <a href="matlab:help LinProp.splineintegrate2 -displayBanner"   >splineintegrate2</a>    Spline integration with scalar result
 %
-% LinProp System (RealUncNumber)
-% a = LinProp(value, [sys_inputs], [sys_sensitivities], 'system')
+% <strong>Object Behavior</strong>
+% Scalar LinProp objects behave like MATLAB fundamental types with respect
+% to copy operations. Copies are independent values. Operations that you
+% perform on one object do not affect copies of that object.
+% Non-scalar LinProp objects are referenced by their handle variable.
+% Copies of the handle variable refer to the same object. Operations that
+% you perform on a handle object are visible from all handle variables that
+% reference that object.
 %
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, idof, id, description)
+% B = <a href="matlab:help LinProp.copy -displayBanner">copy</a>(A) copies each element in the array of handles A to a new
+% array of handles B.
+
+% Metas.UncLib.Matlab.LinProp V2.5.4
```

Comment:
I would like to move the lines 50-52 (author, modification dates) on top of the file.